flujo/application/core/default_cache_components.py

```
@@ -1,389 +1,15 @@
 
-import asyncio
-import hashlib
-import time
-from collections import OrderedDict
-from dataclasses import dataclass, field
-from typing import Protocol
+from typing import TYPE_CHECKING
 
-from ...domain.models import StepResult, UsageLimits
+if TYPE_CHECKING:
+    from flujo.application.core.runtime.default_cache_components import *  # noqa: F401,F403
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)